eluthia/build.py

```diff
@@ -59,6 +69,15 @@ def build_app(app):
         for path, f in flatten_lists(flatten_paths(tree)):
             full_path = (build_folder, package_name, *path)
             f(full_path, *args)
-        
-        if os.environ['SKIP_DEB'].lower() != "true":
-            subprocess.run(["dpkg-deb", "--build", f"{build_folder}/{package_name}"],check=True)
\ No newline at end of file
+
+        if not skip_deb: # Build debian packages and put in zipapp directory
+            subprocess.run(["dpkg-deb", "--build", f"{build_folder}/{package_name}"],check=True)
+            shutil.copy(f'{build_folder}/{package_name}.deb', f'{build_folder}/zipapp/{package_name}.deb')
+
+    # Create executable archive
```

Comment:
I would have only one `if not skip_deb` condition to do both deb and zip packaging, you will have to loop packages again here:
```py
if not skip_deb:
  for package_name, _build in get_builds(os.environ['MACHINE_FOLDER']):
    subprocess.run(['dpkg-deb', ...], check=True)
    shutil.copy(..., ...)
  zipapp.create_archive(...)
  subprocess.run(['chmod', ...], check=True)
```